--- pyinstxtractor.py.orig	2020-10-19 17:28:49.981928563 +0000
+++ pyinstxtractor.py	2020-10-19 17:28:58.250060264 +0000
@@ -311,11 +311,11 @@
             pycHeader = f.read(4) # Python magic value
 
             # Skip PYZ extraction if not running under the same python version
-            if pyc_magic != pycHeader:
-                print('[!] Warning: This script is running in a different Python version than the one used to build the executable.')
-                print('[!] Please run this script in Python{0} to prevent extraction errors during unmarshalling'.format(self.pyver))
-                print('[!] Skipping pyz extraction')
-                return
+#            if pyc_magic != pycHeader:
+#                print('[!] Warning: This script is running in a different Python version than the one used to build the executable.')
+#                print('[!] Please run this script in Python{0} to prevent extraction errors during unmarshalling'.format(self.pyver))
+#                print('[!] Skipping pyz extraction')
+#                return
 
             (tocPosition, ) = struct.unpack('!i', f.read(4))
             f.seek(tocPosition, os.SEEK_SET)
